### PR TITLE
fixed links/formatting/writing edits in vi faq page (Issue #1276)

### PR DESCRIPTION
**Pull request by thesiege208**
rawgit link [here](https://rawgit.com/thesiege208/thesiege208.github.io/step8-issue3/#!pages/vi/vi-faq.md)

### CODE REVIEW

File: pages/vi/vi-faq.md [hunk 1]
```diff
@@ -3,101 +3,101 @@
 ## General Internship Questions
 
 - **Q1:** [What will I be working on during this internship?](#Q1:_What_will_I_be_working_on_during_this_internship?)
-- **Q2:** [What kind of software will we be using for this internship?](#Q2:_What_kind_of_software_will_we_be_using_for_this_internship?)
+- **Q2:** [What kind of software will I be using for this internship?](#Q2:_What_kind_of_software_will_I_be_using_for_this_internship?)
 - **Q3:** [Where can I find more information about this internship?](#Q3:_Where_can_I_find_more_information_about_this_internship?)
-- **Q4:** [What are the hours of the internship?](#Q4:_What_are_the_hours_of_the_internship?)
+- **Q4:** [What is the time commitment for this internship?](#Q4:_What_is_the_time_commitment_for_this_internship?)
```
Comment (ajGingrich) [inline pages/vi/vi-faq.md]: You have changed the meaning of question 4. It should stay as it was originally.

File: pages/vi/vi-faq.md [hunk 2]
```diff
@@ -3,101 +3,101 @@
...
 
 ### General Internship Questions
 
 #### Q1: What will I be working on during this internship?
 
-OLE virtual interns will be helping develop OLE’s BeLL (Basic e-Learning Library) software. The BeLL is a virtual library that is deployed internationally to children in countries that typically do not have access to educational resources.
+OLE virtual interns will be helping develop OLE’s BeLL (Basic e-Learning Library) software. BeLL is a virtual library that is deployed internationally to children in countries that typically do not have access to educational resources.
 
-#### Q2: What kind of software will we be using for this internship?
+#### Q2: What kind of software will I be using for this internship?
```
Comment (ajGingrich) [inline pages/vi/vi-faq.md]: This make sense

**Comment by thesiege208** [comment]
@ajGingrich Fixed, thanks! I also accidentally created this branch from a previous branch regarding other issues - should I remove those changes from this branch to prevent merge conflicts?

**Comment by thesiege208** [comment]
@bsocial Which edits are unnecessary? Also, the branch should be synced with master now, sorry about that.

**Comment by bsocial** [comment]
Hi @thesiege208,

I appreciate that you have made a significant effort with this file. However, the pull request makes too many changes at once making it very difficult to parse, especially when there are conflicts or incorrect edits. For example, we use particular language to describe our product - Nations and Communities. We capitalize these for a reason. I understand that you could not know this. There are other examples of this type of error as well as a few others that are incorrect due to grammar and/or intent. Long story short if these were submitted as individual pull requests we could accept/reject on a case by case basis.

**Comment by thesiege208** [comment]
@bsocial I've split my changes into 2 PRs, one regarding edits to the questions only #1295 and the other regarding edits to the answers only #1299. I think my edits are very straightforward and did not need to be split further than this.

**Comment by kylemathias** [comment]
@thesiege208 I agree with what you are saying. Your Edits are fine, and all necessary. I appreciate you taking the extra steps on fixing things you find wrong along the way. @ajGingrich brings up a good point, if you are making more changes than the original issue described, either make a new issue and pull request to fix those issues or update the original issue to reflect the changes you are going to be making. Im going to close this out as it seems you have done your changes in two different pull request (#1295 #1299)

**Pull request closed without merging**